main.go

```diff
@@ -81,7 +92,60 @@ func main() {
 	fmt.Println()
 	log.Infof("$ apt-get %s", command.PrintableCommandArgs(false, cmdArgs))
 	if err := command.RunCommand("apt-get", cmdArgs...); err != nil {
-		log.Errorf("Can't install packages:  %s", err)
-		os.Exit(1)
+		fail("Can't install packages:  %s", err)
+	}
+}
+
+func applyCacheConfig(cacheConfig string) error {
+	switch cacheConfig {
+	case cacheInputNone:
+		return applyNoCache()
+	case cacheInputAll:
+		return applyAllCache()
+	default:
+		return fmt.Errorf("invalid cache level, no such configuration: %s", cacheConfig)
```

Comment:
this is a not used case because the step logic accepts two value, and it is forced to have one of them selected already.

I think you should remove the constants cacheInputNone and cacheInputAll because a single check
cachelevel == "all" so you would need only one constant anyway